core/src/main/java/org/apache/gravitino/authorization/AuthorizationUtils.java

```diff
@@ -116,4 +123,65 @@ public static void checkRoleNamespace(Namespace namespace) {
         "Role namespace must have 3 levels, the input namespace is %s",
         namespace);
   }
+
+  public static <T> void prepareAuthorizationHooks(T manager, LifecycleHooks hooks) {
+    if (manager instanceof SupportsMetalakes) {
```

Comment:
This method only hooks the event of `createXXXX`?

Reply:
Yes.